atd-etl/cr3_download/README.md

```diff
@@ -3,14 +3,15 @@
 ### Invocation
 
 After creating an .env file using the variables listed in the env_template file, you can run this script with:
+
+If you are running this for the first time or developing this script, you will need to run and build:
 ```
-$ docker compose run -it cr3_download python cr3_download.py
+$ docker compose run --build -it cr3_download python cr3_download.py
```

Comment:
What would you think of adding `CMD python /app/cr3_download.py` as the last line of the `Dockerfile` so that the daily invocation becomes `docker compose run`?